truffle-config.js

```diff
@@ -13,6 +13,27 @@
  */
 
 module.exports = {
+  networks: {
+    development: {
+      host: "localhost",
+      port: 8545,
+      network_id: "*"
+    }
+  },
   // See <http://truffleframework.com/docs/advanced/configuration>
   // to customize your Truffle configuration!
+  // Configure your compilers
+  compilers: {
+    solc: {
+      version: "v0.6.2",    // Fetch exact version from solc-bin (default: truffle's version)
+      // docker: true,        // Use "0.5.1" you've installed locally with docker (default: false)
+      // settings: {          // See the solidity docs for advice about optimization and evmVersion
+      //  optimizer: {
+      enabled: true,
```

Comment:
Don't you have to comment out the settings and optimizer line as well for this to have an effect? 
Also just a nit, but I'd slightly prefer if we clean up all the unneeded code.